include/swift/Basic/Platform.h

```diff
@@ -22,6 +22,17 @@ namespace llvm {
 }
 
 namespace swift {
+
+  enum class DarwinPlatformKind : unsigned {
```

Comment:
These names match the names clang uses. I think keeping the same names will simplify code reuse.